crates/ruff_python_stdlib/src/identifiers.rs

```diff
@@ -1,6 +1,13 @@
+use crate::keyword::KWLIST;
+
 /// Returns `true` if a string is a valid Python identifier (e.g., variable
 /// name).
 pub fn is_identifier(s: &str) -> bool {
+    // Is the string a keyword?
+    if KWLIST.contains(&s) {
+        return false;
+    }
+
     // Is the first character a letter or underscore?
     if !s
```

Comment:
Unrelated to this PR but this could be rewritten to

```suggestion
let mut chars = s.chars();

if !s.next().map_or(false, |c| is.alphabetic() || c == '_') {
	return false;
}

chars.all(|c| c.is_alphanumeric() || c == '_')
```

to avoid iterating over the first char twice.